call_graph.html

```diff
@@ -0,0 +1,530 @@
+<!--
```

Comment:
The line numbers defined in the call graph don't match actual line numbers in their python source code file. For instance, `set_title (__main__.py:271)` should be `set_title (__main__.py:278)`. 

Beyond that, if the call graph is going to be added/committed, shouldn't it be automatically generated as part of the CI process? That way the graph will always be up-to-date.

Reply:
Changed to generate call graph artifact. Should I remove the file from the repo?

Reply:
Correct, the file should not be committed in the repo if it is being generated as an artifact.

Reply:
8786301b8958b5871f55035e48fd6c9a964a38cc